lib/template_requirements_config.yaml

```diff
@@ -0,0 +1,11 @@
+test_kit_id: # the name of the repository slug. E.g., onc-certification-g31-test-kit
```

Comment:
FYI - this is changing in a future version to the underscore version (Steve set me straight)

Reply:
I updated the comment here to reference the underscored name.